NCLCoreClasses/Camera.cpp

```diff
@@ -39,6 +39,16 @@ void Camera::UpdateCamera(float dt) {
 
 }
 
+Vector3 Camera::GetForwardVector() {
+	Matrix4 yawRotation = Matrix4::Rotation(yaw, Vector3(0, 1, 0));
```

Comment:
This will work fine, but you can also get the forward vector by taking one of the columns of the Matrix4 - I can't remember which! But might save us a matrix mult :)